public/app/features/playlist/types.ts

```diff
@@ -1,16 +1,8 @@
+import { Playlist } from '../../api/clients/playlist';
 import { DashboardQueryResult } from '../search/service/types';
+export type PlaylistMode = boolean;
 
-export type PlaylistMode = boolean | 'tv';
-
-export interface PlaylistAPI {
-  getAllPlaylist(): Promise<Playlist[]>;
-  getPlaylist(uid: string): Promise<Playlist>;
-  createPlaylist(playlist: Playlist): Promise<void>;
-  updatePlaylist(playlist: Playlist): Promise<void>;
-  deletePlaylist(uid: string): Promise<void>;
-}
-
-export interface Playlist {
+export interface PlaylistUI {
```

Comment:
why `PlaylistUI` and not the out-of-the-box PlaylistSpec?

Reply:
Agree, can we use the generated types directly instead of creating aliases? 🙏  Makes it easier to look up.